common/src/main/java/com/revenuecat/purchases/common/caching/DeviceCache.kt

```diff
@@ -129,7 +129,8 @@ open class DeviceCache(
     fun cacheCustomerInfo(appUserID: String, info: CustomerInfo) {
         val jsonObject = info.rawData.also {
             it.put(CUSTOMER_INFO_SCHEMA_VERSION_KEY, CUSTOMER_INFO_SCHEMA_VERSION)
-            it.put(CUSTOMER_INFO_VERIFICATION_RESULT_KEY, info.entitlements.verification.name)
+            // Trusted entitlements: Commented out until ready to be made public
+            // it.put(CUSTOMER_INFO_VERIFICATION_RESULT_KEY, info.entitlements.verification.name)
```

Comment:
I'm not a fan of changing the behavior like this for this... But since `verification` is `internal` in `EntitlementInfos` and `EntitlementInfo`, we can't do this. 

So basically we should test the behavior thoroughly for this PR, basically everything should default to disabled.